lib/convertor.js

```diff
@@ -229,7 +229,9 @@ class MongoToKnex {
                                 });
                             });
 
-                        debug(`(buildRelationQuery) innerQB sql-pre: ${innerQB.toSQL().sql}`);
+                        if (debugExtended.enabled) {
```

Comment:
These if's are quite ugly and might be a good idea to extract them into separate debugExtended() method for better readability

Reply:
seems like a good idea (again for later)